relayer/pkg/chainlink/relay.go

```diff
@@ -51,9 +51,7 @@ func (r *relayer) Ready() error {
 	return r.chainSet.Ready()
 }
 
-func (r *relayer) Healthy() error {
-	return r.chainSet.Healthy()
-}
+func (r *relayer) Healthy() error { return nil }
```

Comment:
Any reason why we still need this and not remove all of it?

Reply:
same question for solana

Reply:
I was just trying to do the minimum to fix things and thought that might create more of the same problem :shrug: